app/views/categories/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<p style="color: green"><%= notice %></p>
+<h1>Categories</h1>
+<div id="categories">
+  <% @categories.each do |category| %>
+    <%= link_to category_path(category) do %>
```

Comment:
[OPTIONAL] Please ensure the current user can see only their own categories. I sign up with another account and I can still the categories of another user.